pkg/core/basic_chain_test.go

```diff
@@ -593,3 +622,1630 @@ func TestBlockchain_StartFromExistingDB(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestBlockchain_VerifyHeader(t *testing.T) {
+	bc, acc := chain.NewSingleWithCustomConfig(t, func(c *config.ProtocolConfiguration) {
+		c.StateRootInHeader = true
+	})
+	e := neotest.NewExecutor(t, bc, acc, acc)
+
+	newHeader := func(t *testing.T, index uint32, prevHash util.Uint256, timestamp uint64) *block.Header {
+		b := e.NewUnsignedBlock(t)
+		b.Index = index
+		b.PrevHash = prevHash
+		b.Timestamp = timestamp
+		e.SignBlock(b)
+		return &b.Header
+	}
+	prev := e.TopBlock(t).Header
+	t.Run("Invalid", func(t *testing.T) {
+		t.Run("Hash", func(t *testing.T) {
+			// TODO: can't port this test.
```

Comment:
Then `VerifyHeader` shouldn't be ported at all probably. Creating bad headers/blocks is out of scope for `neotest`, so it's OK to have something special for them.

Reply:
Fixed.